### DISCUSSION TITLE
the effect of tissue segmentation on e-field calculation

### DISCUSSION THREAD

**Discussion by windyjunfeng** [simnibs/simnibs#403] [8. Simulation workflows] [answered]
Hi, I wonder if I can only segment gray matter, white matter and cerebrospinal fluid (not including scalp, bone and other tissues) and calculate the e-field. I only care about the e-field of these three tissues. Is that right in theory? Thank you very much!

**Comment by oulap** [comment, marked as answer]
Hi,

Even if you only care about the e-fields in those tissues, I assume you still want the simulation to reflect an anatomy where all tissues are present. The e-field has to propagate through tissues in between the stimulator and the brain, such as the scalp and the skull, which affect the e-field distribution in the whole head. 

However, if you actually do want to stimulate the brain directly (without the scalp and skull) you can do that in SimNIBS but it requires a fair bit of extra hacking. I'm not sure though what you would want to achieve with that type of simulation.

**Reply by windyjunfeng** [reply]
Thank you very much. I got it. I found that the e-field distribution of gray matter would change slightly when the electrical conductivity of scalp and skull changed. So, I shouldn't ignore the effect of scalp and skull.
